### ISSUE TITLE
Photograph items are missing alt text

### ISSUE THREAD

**Issue by erinchambers** [CDRH/family_letters#205]
Item images for the photographs category have captions just above them, but adjacent text doesn't count toward the WCAG 2.1 alt text criteria. If it turns out that the captions don't describe the images very well, we can address that through #202. If there are images that don't have captions, we'll have to add alt text for them before the deadline.

As it stands, I propose using the same text for the image `alt` text and the caption, then adding `aria-hidden="true"` to the caption so that screen readers don't read the caption, just the alt text. That'll keep it from being duplicated for assistive technology users. 

``` html
# Current code
<div class="image_item_display">
    <p>Black and white portrait of Barbara Fe in a coat and shirt with pendant</p>
    <img src="https://cdrhmedia.unl.edu/iiif/2/family_letters%2Fshan_p.063.jpg/full/!800,800/0/default.jpg" class="display">
  </div>

# Proposed changes
<div class="image_item_display">
    <p aria-hidden="true">Black and white portrait of Barbara Fe in a coat and shirt with pendant</p>
    <img alt="Black and white portrait of Barbara Fe in a coat and shirt with pendant" src="https://cdrhmedia.unl.edu/iiif/2/family_letters%2Fshan_p.063.jpg/full/!800,800/0/default.jpg" class="display">
  </div>
```

**Comment by erinchambers** [comment]
I figured out that the code would go in [file_csv.rb](https://github.com/CDRH/data_family_letters/blob/f3ab31e17d00235eddd819652c95d3f79f176c53/scripts/overrides/file_csv.rb#L20) but then hit a wall trying to write code... I'm not great at ruby scripts :/

**Comment by erinchambers** [comment]
Figured it out - PR ready
